<?xml version="1.0" encoding="UTF-8" ?>
<testsuites errors="0"
            failures="1"
            tests="1">

  <testsuite name="test_wallet"
    errors="0" failures="1" hostname="" id="" package="test_wallet" skipped="0"
    tests="1" time="10.83" timestamp="">
  
    <testcase name="navigate to Wallets" classname="test_wallet" time="10.83" assertions="14">

    

    

    

    

    

    

    

    

    

    

    

    

      <failure message="Failed [equal]: (description for row count assert) - expected [0;32m&#34;7&#34;[0m but got: [0;31m&#34;1&#34;[0m">    at NightwatchAPI.&lt;anonymous&gt; (C:\Users\DIT\Desktop\project\bankup-front-web2\app\src\tests\test_wallet.js:34:21)
    at process._tickCallback (internal/process/next_tick.js:68:7)</failure>

    <failure message="    at NightwatchAPI.&lt;anonymous&gt; (C:\Users\DIT\Desktop\project\bankup-front-web2\app\src\tests\test_wallet.js:34:21)">    at process._tickCallback (internal/process/next_tick.js:68:7)</failure>
    </testcase>
  

  

  
  </testsuite>
</testsuites>
